android/ui/src/main/java/com/trafi/ui/theme/Color.kt

```diff
@@ -117,6 +113,8 @@ class MaasColorPalette(
         private set
     var onError by mutableStateOf(onError)
         private set
+    var grayScale by mutableStateOf(grayScale)
```

Comment:
What do you think about calling this `gray` and the various variants just `100`, `200`, etc? So we could write `MaasTheme.colors.gray.100` 🤔

Reply:
🔝 good idea 🤔

Reply:
😮 what if wrapped in backticks, e.g.
```
val `100` = ..
```
Not pretty though, so probably not the way to go